app/domain/authentication/authn_k8s/validate_application_identity.rb

```diff
@@ -10,12 +10,12 @@ module AuthnK8s
 
     ValidateApplicationIdentity = CommandClass.new(
```

Comment:
Fix indentation of this file. It seems that the only change is that you changed `Role` to `::Role`. Make it look so.